0001-template.md

```diff
@@ -82,7 +82,13 @@ Why should we *not* do this?
 ## Prior art
 [prior-art]: #prior-art
 
-Discuss prior art, both the good and the bad, in relation to this proposal.
+Discuss prior art, both the good and the bad, in relation to this proposal. 
+
+Prior art is any evidence that your feature (invention, change, proposal) is already known. Examples include:
```

Comment:
these examples are too coupled to Rust so they don't make sense in our context. I'd suggest:

```
Thinks about the lessons from other blockchain projects or similar updates and provide readers of your RFC with a fuller picture. If there is no prior art, that is fine. Your ideas are interesting whether they are new or adapted from another source.
```

and then just skip that inner section